### ISSUE TITLE
Make C-u work like in shells/readline, remove C-d

### ISSUE THREAD

**Issue by osa1** [osa1/tiny#300]
Currently `C-u` and `C-d` scrolls up and down, respectively. These keys were copied from vim (normal mode).

However we already have other keys for this: page up and down, and `C-u` removes parts before the cursor in the current line in bash, readline, emacs, and vim insert mode.

So I think it makes sense to make `C-u` consistent with the other tools, and remove `C-d`. Less stuff to learn.

**Comment by Kabouik** [comment]
Do you plan on implementing alternatives other than `PGDN` and `PGUP ` for page scrolling? Not all keyboards have those keys, and visual scrolling using mouse wheel or touch screens doesn't work in things like `mosh`.

**Comment by osa1** [comment]
@Kabouik shift+up/down also scrolls, but just one line instead of one "page". Does that combination work in `mosh`?

**Comment by Kabouik** [comment]
Yes those combination because they rely on Tiny's scrollback feature, not mosh's (which doesn't really allow scrolling by itself, see [#122](https://github.com/mobile-shell/mosh/issues/122)), but one line at a time can be an issue sometimes. Maybe something like `Ctrl+Shift+Up` could work? No idea if that is standard though, probably not, and it may be conflicting with other things.
